geode-core/src/main/java/org/apache/geode/cache/client/internal/ProxyRegion.java

```diff
@@ -187,7 +187,7 @@ public void destroyRegion(Object callbackArgument) throws CacheWriterException,
   public Set entries(boolean recursive) {
```

Comment:
AdminRegion was another class that implements Region that should be updated to no longer impl the methods you are removing. You should look at all the classes that implement Region.